src/app/gm-tools/feature/lib.routes.ts

```diff
@@ -185,6 +186,23 @@ const gmToolsRoutes: Routes = [
     loadComponent: () => import('./new-event-page.component'),
     data: { pageAnimationLayer: 2 },
   },
+  {
+    path: 'events/edit/:eventId',
+    loadComponent: () => import('./edit-event-page.component'),
+    data: { pageAnimationLayer: 2 },
+    resolve: { originalEvent: editEventResolver },
```

Comment:
Look at you using resolvers. They grow up so fast...

Reply:
Don't worry, I'll also be your little boy Dad!

Reply:
S-tier PR comment